src/clj/puppetlabs/services/ca/certificate_authority_service.clj

```diff
@@ -27,9 +30,9 @@
              comidi/routes->handler)
          settings
          path
-         wrap-with-authorization-check
-         puppet-version)))
-   context)
+         (fn [request] (wrap-with-authorization-check request {:oid-map oid-mappings}))
```

Comment:
Cool, if you too are good with that approach then I'm :+1:. I'll get to work on that.